src/Install/Prerequisite/WritablePaths.php

```diff
@@ -53,7 +53,7 @@ private function getNonWritablePaths(): Collection
             })->map(function ($path, $index) {
                 return [
                     'message' => 'The '.$this->getAbsolutePath($path).' directory is not writable.',
-                    'detail' => 'Please chmod this directory'.(in_array($index, $this->wildcards) ? ' and its contents' : '').' to 0775.'
+                    'detail' => 'Please make sure your web server/PHP user has write access to this directory'.(in_array($index, $this->wildcards) ? ' and its contents' : '').'. Please read the <a href="https://docs.flarum.org/install.html#folder-ownership">installation documentation</a> for a detailed explanation and steps to follow to resolve this error.'
```

Comment:
Minor wording critiques:

- Both sentences start with please, that's a bit much imo
- "steps to follow to resolve this error" => "steps to resolve this error"

Reply:
Updated with both changes - looks better now.